sdk/digitaltwins/Azure.DigitalTwins.Core/tests/prerequisites/setup.ps1

```diff
@@ -52,11 +53,6 @@ Connect-AzureSubscription
 
 $Region = $Region.Replace(' ', '')
 
-if (-not $DigitalTwinName)
```

Comment:
is this check required? `DigitalTwinName` being mandatory now, an error will already have been thrown if it isn't set

Reply:
that's why I removed it no?

Reply:
oh yes you did, my bad :dizzy_face:
I need to go back to sleep.